lib/ValueAnonymizers/Phone.php

```diff
@@ -26,7 +26,16 @@ public function anonymize(Value $value, array $row): AnonymizedValue
             return AnonymizedValue::fromOriginalValue($value);
         }
 
-        $anonymizedEscapedValue = $this->stringHash->hashKeepFormat($value->getUnEscapedValue());
+        $string = $value->getUnEscapedValue();
+
+        if (strlen($string) < 10) {
+            $hash = base_convert($this->stringHash->sha256($string), 16, 10);
+            $toAnonymize = '+'.substr($hash, 0, 11);
```

Comment:
I think adding a plus here is too arbitrary.

I think a better option here would be to add digit chars from the decimal hash to the original string until it reaches 10 characters.
Maybe this can be used for the others as well...